docs/ndndpdk-depends.sh

```diff
@@ -282,66 +282,66 @@ if [[ $CONFIRM -ne 1 ]]; then
   read -p 'Press ENTER to continue or CTRL+C to abort '
 fi
 
-$SUDO mkdir -p $CODEROOT
-$SUDO chown -R $(id -u):$(id -g) $CODEROOT
+$SUDO mkdir -p "$CODEROOT"
+$SUDO chown -R "$(id -u):$(id -g)" "$CODEROOT"
 
 echo 'Dpkg::Options {
    "--force-confdef";
    "--force-confold";
 }
 APT::Install-Recommends "no";
 APT::Install-Suggests "no";' | $SUDO tee /etc/apt/apt.conf.d/80custom >/dev/null
-if [[ $DISTRO == 'bionic' ]] && ! [[ -f /etc/apt/sources.list.d/llvm-11.list ]]; then
-  curl -sfL ${NDNDPDK_DL_LLVM_APT}/llvm-snapshot.gpg.key | $SUDO apt-key add -
+if [[ $DISTRO == bionic ]] && ! [[ -f /etc/apt/sources.list.d/llvm-11.list ]]; then
+  curl -sfL "${NDNDPDK_DL_LLVM_APT}/llvm-snapshot.gpg.key" | $SUDO apt-key add -
   echo "deb ${NDNDPDK_DL_LLVM_APT}/bionic/ llvm-toolchain-bionic-11 main" | $SUDO tee /etc/apt/sources.list.d/llvm-11.list
 fi
 $SUDO apt-get -y -qq update
 $SUDO sh -c 'DEBIAN_FRONTEND=noninteractive apt-get -y -qq dist-upgrade'
 
-if [[ $NODEVER != '0' ]]; then
-  curl -sfL ${NDNDPDK_DL_NODESOURCE_DEB}/setup_${NODEVER} | $SUDO bash -
+if [[ $NODEVER != 0 ]]; then
+  curl -sfL "${NDNDPDK_DL_NODESOURCE_DEB}/setup_${NODEVER}" | $SUDO bash -
   APT_PKGS+=(nodejs)
 fi
 
 APT_PKG_LIST="${APT_PKGS[@]}"
 $SUDO sh -c "DEBIAN_FRONTEND=noninteractive apt-get -y -qq install ${APT_PKG_LIST}"
```

Comment:
I wanted to get rid of the `APT_PKG_LIST` intermediate variable, but then I noticed this strange line... why are you using `sh -c` to run apt-get?

Reply:
This is learned from https://get.docker.com script.

Reply:
but why?

Reply:
No particular reason from me.
I simply followed existing examples.